src/context/FilterProjectsContext.jsx

```diff
@@ -245,11 +319,15 @@ export const FilterProjectsProvider = ({ children }) => {
             ? userIsMemberOfProject(project.project_id, mermaidUserData)
             : true
 
+          // Filter out projects that have no records
+          const projectHasRecords = isAnyActiveFilters() && !project.records.length ? false : true
```

Comment:
Is the ternary necessary here? Wouldn't `isAnyActiveFilters() && !project.records.length` result in a boolean anyway?

Reply:
The ternary isn't necessary. And that would result in a boolean anyway but I have to change the logic to `const projectHasRecords = !isAnyActiveFilters() || project.records.length > 0` so that it has the same functionality